src/tutorial/runtime/introduction.rst

```diff
@@ -291,60 +275,161 @@ Where Do All The Files Go?
 
          ~/cylc-run/<workflow-name>/log/job/<cycle-point>/<task-name>/<job-submission-num>/
 
-      .. image:: ../img/cylc-gui-view-log.png
+      .. TODO REPLACE THIS IF APPROPRIATE: .. image:: ../img/cylc-gui-view-log.png
          :align: center
          :scale: 75%
 
 
-Running A Workflow
-------------------
+Installing A Workflow
+---------------------
 
 .. ifnotslides::
 
-   It is a good idea to check a workflow for errors before running it.
-   Cylc provides a command which automatically checks for any obvious
-   configuration issues called ``cylc validate``, run via:
+   .. seealso::
+
+      :ref:`The full guide to Cylc install <Installing-workflows>`.
+
+   .. versionchanged:: 8.0.0
+
+      ``cylc install`` is a new feature of Cylc 8.
+
+   To allow you to separate the development and running of workflows
+   Cylc provides a :term:`cylc install <install>` command.
+
+For a workflow developed in
+
+- ``~/cylc-src/my_workflow`` and
+- ``/some/location/my_other_workflow``
+
+respectively:
+
+.. code-block:: shell
```

Comment:
FYI `shell` is the same as `bash` which is more commonly used in the docs.

Reply:
Changed for consistency.